Makefile

```diff
@@ -92,7 +92,7 @@ libbedrock.a: $(LIBBEDROCKOBJ)
 
 # We use the same library paths and required libraries for both binaries.
 LIBPATHS =-Lmbedtls/library -L$(PROJECT)
-LIBRARIES =-lbedrock -lstuff -ldl -lpcrecpp -lpthread -lmbedtls -lmbedx509 -lmbedcrypto -lz
+LIBRARIES =-lbedrock -lstuff -lbedrock -ldl -lpcrecpp -lpthread -lmbedtls -lmbedx509 -lmbedcrypto -lz
```

Comment:
im confused, `-lbedrock` is the first library in this list of args already

Reply:
It needs to be in both places because now that `getState` isn't in the header, SHTTPSManager, which is where it's called to see if we can run https requests, needs to link against it, and dependencies always have to come last in this list. We can't just reverse the order of `-lbedrock` and `-lstuff`, because a bunch of stuff in bedrock depends on libstuff already, so we add it a second time so that libstuff knows it needs to look at libbedrock again.